backend/config/initializers/sentry.rb

```diff
@@ -0,0 +1,13 @@
+Sentry.init do |config|
+    config.dsn = ENV["SENTRY_KEY"]
+    config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+  
+    # Set tracesSampleRate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production
+    config.traces_sample_rate = 1.0
+    # or
+    # config.traces_sampler = lambda do |context|
+    #   true
+    # end
+  end
```

Comment:
Add empty line (maybe you should config this on your IDE or if possible we could add this rule torubocop)

Reply:
solved!